### PR TITLE
Remove `eslint-plugin-prettier`

### PR DESCRIPTION
**Pull request by fisker**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | `Fixes #1, Fixes #2` 
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

It's bad to run Prettier as ESLint plugin, it's super slow.

- [Why eslint-plugin-prettier Is Often Harmful](https://www.joshuakgoldberg.com/blog/you-probably-dont-need-eslint-config-prettier-or-eslint-plugin-prettier/#why-eslint-plugin-prettier-is-often-harmful)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -111,10 +111,14 @@
   },
   "lint-staged": {
     "./Makefile.source.mjs": [
-      "node ./scripts/pack-script.js --auto"
+      "node ./scripts/pack-script.js --auto",
+      "git add Makefile.js"
```
Comment (nicolo-ribaudo) [inline package.json:115]: Thanks :)

File: .prettierignore [hunk 2]
```diff
@@ -2,12 +2,31 @@ package.json
...
+packages/babel-parser/typings/babel-parser.d.ts
 Makefile.js
+
+# Should enable?
+packages/babel-runtime-corejs2/helpers/**
+packages/babel-runtime-corejs2/core-js/**
+packages/babel-runtime-corejs3/helpers/**
+packages/babel-runtime/helpers/**
+**/tsconfig.json
+**/tsconfig.*.json
```
Comment (nicolo-ribaudo) [inline .prettierignore:26]: These files are all generated

File: .prettierignore [hunk 3]
```diff
@@ -2,12 +2,31 @@ package.json
...
+*.sublime-project
```
Comment (nicolo-ribaudo) [inline .prettierignore:27]: @liuxingbaoyu @JLHwung do any of you use sublime text? If not, let's in a separate PR just delete this file.

File: .prettierignore [hunk 4]
```diff
@@ -2,12 +2,31 @@ package.json
...
+**/*.yml
+**/*.md
+packages/babel-parser/test/schema.json
+.c8rc
+test/runtime-integration/node.cjs
```
Comment (nicolo-ribaudo) [inline .prettierignore:32]: These three can be formatted.

**Review by nicolo-ribaudo — APPROVED** [review]
Thank you!

This looks good. I left a few comments but if you prefer I can just push a commit to handle them.

File: .prettierignore (hunk 4 above)
Comment (fisker) [reply .prettierignore]: Did you mean

```
packages/babel-parser/test/schema.json
.c8rc
test/runtime-integration/node.cjs
```

?

The other two moved to top, but `.c8rc` should able to format.

**Comment by fisker** [comment]
> I can just push a commit to handle them.

I think you can decide which file to format and format them in this PR.

**Comment by fisker** [comment]
I also suggest adding similar settings like https://github.com/prettier/prettier/blob/aa3853b7765645b3f3d8a76e41cf6d70b93c01fd/.vscode/settings.example.json#L2-L10

to .vscode/settings.example.json

File: .prettierignore (hunk 3 above)
Comment (JLHwung) [reply .prettierignore:29]: Although I don't use sublime, I think it is better to keep it for the convenience of [babel-sublime](https://github.com/babel/babel-sublime) maintainers.

**Review by JLHwung — APPROVED** [review]
Thanks!

**Pull request merged**